Memstate.Docs.GettingStarted/QuickStart/Commands/InitCustomer.cs

```diff
@@ -13,7 +14,10 @@ public InitCustomer(int id, int points)
             Points = points;
         }
 
+        [JsonProperty("ID")]
```

Comment:
Tests will fail without the `JsonProperty` if you use the `JsonSerializerAdapter` as it currently stands. This is the only way that anyone can use `memstate` in it's current form. If you change memstate, I'll be happy to update the tests. That's exactly why I've written the tests inside the Docs, so that we can gaurantee that the getting started instructions actually work.

Reply:
You really need the latest code... there are a lot of changes